### PR TITLE
Update Root Readme

### PR DESCRIPTION
**Pull request by JoeOster**
# Description

Updating root readme.md

Fixes # (issue) 

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Sample Migration (Moving sample from old repository after completing checklist established)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode

# Checklist for Moving samples:
Links and Details can be found in the samples WG Teams Files. 

- [ ] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
- [ ] Implement coding guidelines and ensure code quality.
- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
- [ ] Adhere to readme template 
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ ] Enforce format via clang-format config file
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth or Joe Oster. (GitHub User: JoeOster)
- [ ] Tested using Dev Cloud when applicable
- [ ] Implement fixes for ONSAM Jiras
- [ ] If you have new dependencies/binaries, inform Samples Project Manager Swapna R Dontharaju (@srdontha) or @JoeOster

### CODE REVIEW

**Comment by akertesz** [comment]
Is this an accurate representation of how this will look?
https://github.com/JoeOster/oneAPI-samples

If so, the formatting is wonky. The rightmost table column is much too narrow. Also, the titles of the sections at the bottom aren't rendering correctly. Not sure what's wrong there.

**Review by mkitez — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+  
+    1. Select the "General" tab on the left,
+    2. Select on the right side of the dialog box "Windows SDK Version"(2nd Item".
+    3. Click on the drop-down icon to select a version that is installed on your system.
+    4. click on [Ok] to save.
+
+3. Now you should be able to build the code sample.
+4. For beta, FPGA samples support Windows through FPGA-emulator.
+5. If you encounter a compilation error like below when building a sample program, one reason is that the directory path of the sample is too long; the work around is to move the sample to a directory like "c:\temp\sample_name".
+  * Error MSB6003 The specified task executable "dpcpp-cl.exe" could not be run ......
```
Comment (sravanikonda) [inline README.md:84]: dpcpp-cl.exe changed to dpcpp.exe

File: README.md [hunk 2]
```diff
@@ -1,2 +1,85 @@
...
+    2. Right-click on the project name in "Solution Explorer" and select "Properties"
+  
+  2. The project property dialog opens. 
+  
+    1. Select the "General" tab on the left,
+    2. Select on the right side of the dialog box "Windows SDK Version"(2nd Item".
+    3. Click on the drop-down icon to select a version that is installed on your system.
+    4. click on [Ok] to save.
+
+3. Now you should be able to build the code sample.
```
Comment (sravanikonda) [inline README.md:81]: This can be removed too, looks out of place

File: README.md [hunk 3]
```diff
@@ -1,2 +1,85 @@
...
+
+#License
+ 
+The code samples are licensed under MIT license
+ 
+#Known issues or limitations
+ 
+##On Windows Platform
+1. If you are using Visual Studio 2019, Visual Studio 2019 version 16.3.0 or newer is required.
+2. To build samples on Windows, the required Windows SDK is ver. 10.0.17763.0.
```
Comment (sravanikonda) [inline README.md:67]: This whole bullet can be removed as we no longer depend on specific SDK versions

File: README.md [hunk 4]
```diff
@@ -1,2 +1,85 @@
...
+|../SystemDebug/System Debug   Sample Build    	|UEFI 	|Basic example that   showcases the features of the Intel® System Debugger	|
+
+#License
+ 
+The code samples are licensed under MIT license
+ 
+#Known issues or limitations
+ 
+##On Windows Platform
+1. If you are using Visual Studio 2019, Visual Studio 2019 version 16.3.0 or newer is required.
```
Comment (sravanikonda) [inline README.md:66]: Could you change this to 16.4.0?

**Comment by anjgola** [comment]
Pending on changes requested by @sravanikonda

File: README.md (hunk 3 above)
Comment (anjgola) [reply README.md:67]: @JoeOster  you marked this as resolved but the change has not been made. Can you confirm that this is indeed resolved.

**Pull request merged**